auth.go

```diff
@@ -159,10 +183,11 @@ func (s *slackAuth) runServer() error {
 	return srv.ListenAndServe()
 }
 
-func (s *slackAuth) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+func (s *slackAuth) authorizationHandler(w http.ResponseWriter, r *http.Request) {
 	code := r.FormValue("code")
 	resp, err := s.api.GetOAuthResponse(s.clientID, s.clientSecret, code, s.debug)
 	if err != nil {
+		w.WriteHeader(409)
```

Comment:
why statusconflict? also, it's better to return status codes with the constants in `net/http`, for example `http.StatusOK` rather than integers